package.json

```diff
@@ -12,14 +12,27 @@
     },
     "./dist/": "./dist/",
     "./icons": "./dist/esm/src/components/Icon/Icons/icons.js",
-    "./tokens": "./dist/tokens/tokens.css",
-    "./storybookComponents": "./dist/storybook/index.js",
     "./interactionsTests": "./dist/esm/src/tests/interactionsTests.js",
     "./testIds": "./dist/esm/src/tests/testIds.js",
+    "./storybookComponents": "./dist/storybook/index.js",
+    "./tokens": "./dist/tokens/tokens.css",
     "./mockedClassNames": {
       "import": "./dist/mocked_classnames_esm/src/index.js"
     }
   },
+  "typesVersions": {
+    "*": {
+      "icons": [
+        "./dist/esm/components/Icon/Icons/index.d.ts"
+      ],
+      "interactionsTests": [
+        "./dist/esm/tests/interactions-utils.d.ts"
+      ],
+      "testIds": [
+        "./dist/esm/tests/test-ids-utils.d.ts"
+      ]
```

Comment:
Why are these two needed to be publically exposed APIs?
The `interactionsTests` are mostly wrappers from regular DOM functions and they are very platform specific

Reply:
`testIds` are containing `ComponentDefaultTestId` which contains all default data-testid for components and it's being used already in the monolith in cypress tests selectors
<img width="1210" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/eac62491-e107-4215-9c1b-fa9f49d79501">

`interactionsTests` are exporting functions which helps to build storybook interaction tests and this endpoint is being used in the `monday-ui-components` rn
<img width="941" alt="image" src="https://github.com/mondaycom/monday-ui-react-core/assets/104433616/c8ea42c2-7050-44da-a832-9054b05b67a6">

Also third one should be `storybookComponents` to export components we are using to build our storybook, but as they are not converted to ts, they are not here

Reply:
ok. I'm not too fond of it as a public API. it creates a coupling between Vibe's internal testing tools with others who consume it, making it harder to refactor. if that's the current state, then fine, but let's try to reduce such dependencies